src/logic/basic.lean

```diff
@@ -242,9 +242,17 @@ mt imp_intro
 
 theorem dec_em (p : Prop) [decidable p] : p ∨ ¬p := decidable.em p
 
-theorem em (p : Prop) : p ∨ ¬ p := classical.em _
+theorem dec_em' (p : Prop) [decidable p] : ¬p ∨ p := (dec_em p).swap
 
-theorem or_not {p : Prop} : p ∨ ¬ p := em _
+theorem em (p : Prop) : p ∨ ¬p := classical.em _
+
+theorem em' (p : Prop) : ¬p ∨ p := (em p).swap
```

Comment:
I don't really see the point in this one - why not just let the caller add the `swap`?

Reply:
I wasn't originally going to add that one but [Mario thought it might be worthwhile](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/eq_or_ne/near/237029694).